spec/acceptance/mongodb_exporter_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper_acceptance'
+
+describe 'prometheus mongodb_exporter' do
+  it 'mongodb_exporter works idempotently with no errors' do
+    pp = 'include prometheus::mongodb_exporter'
+    apply_manifest(pp, catch_failures: true)
+    apply_manifest(pp, expect_changes: true)
+    shell('sleep 30')
+  end
+
+  describe 'prometheus mongodb_exporter version 0.3.1' do
+    it ' mongodb_exporter installs with version 0.3.1' do
+      pp = "class { 'prometheus::mongodb_exporter': version => '0.3.1' }"
+      apply_manifest(pp, catch_failures: true)
+      apply_manifest(pp, catch_changes: true)
+      shell('sleep 30')
+    end
+
+    describe process('mongodb_exporter') do
+      its(:args) { is_expected.to match %r{\ -mongodb.uri} }
+    end
```

Comment:
Something I generally don't like about serverspec tests is that if they fail, it's very hard to debug. It looks like `ps -C mongodb_exporter` doesn't return anything. You could try to use:
```suggestion
    describe command('pgrep mongodb_exporter') do
      its(:stdout) { is_expected.to match %r{\ -mongodb.uri} }
    end
```

Reply:
This did not work as expected: 
* https://travis-ci.org/github/voxpupuli/puppet-prometheus/builds/691675038